app/src/components/inputs/inputDropdown/inputDropdownOption/inputDropdownOption.jsx

```diff
@@ -25,39 +25,43 @@ import styles from './inputDropdownOption.scss';
 
 const cx = classNames.bind(styles);
 
-export const DropdownOption = ({ multiple, text, disabled, active }) => {
+
+export const DropdownOption = ({ multiple, text, disabled, active, onChange, id }) => {
   const dropdownOptionClasses = cx({
     'dropdown-option': true,
     active: !multiple && active,
     disabled,
   });
+  const onChangeHandler = (checkedObj) => {
+    onChange(Object.assign(checkedObj, { value: id }));
+  };
   return (
     <div className={dropdownOptionClasses}>
       {
         multiple
-          ? <InputCheckbox value={active} disabled={disabled} >{text}</InputCheckbox>
-          : <div className={cx('single-option')}>{text}</div>
+          ? <InputCheckbox value={active} disabled={disabled} onChange={onChangeHandler}>
+            {text}
+          </InputCheckbox>
+          : <div className={cx('single-option')} onClick={() => onChange(id)}>{text}</div>
```

Comment:
In case of `multiple` you call `onChange` with object, and here you call it with `id` only. This should be consistent.